staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -18,8 +18,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../../../../..
-EXAMPLE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=${GOPATH}/src/k8s.io/kubernetes
```

Comment:
To make this reusable and easier to read: what about turning the generator calls into one big bash functions that gets GROUP_NAME, VERSION as parameters:
```
generate_group wardle v1alpha1
```
Then only BASE_PATH and BASE_PKG would be global variables. The other *_PKG variables can become local variables in the generate_group func.